libraries/shared/test_data/test_helpers.go

```diff
@@ -88,3 +89,30 @@ func getRandomAddress() string {
 	stringHash := hashToPrefixedString(hash)
 	return stringHash[:42]
 }
+
+/// DescribeAValidCheckedHeadersModelWithSchema is provided so that
+/// plugins can easily validate that they have the necessary checked_headers
+/// table in their schema for the execute process.
+///
+/// Use like so in your tests:
+/// var _ = test_data.DescribeAValidCheckedHeadersModelWithSchema("schemaName")
+func ExpectCheckedHeadersInThisSchema(db *postgres.DB, schema string) {
+	Expect(db).NotTo(BeNil())
+	// insert header
+	blockNumber := rand.Int63()
+	fakeHeader := fakes.GetFakeHeader(blockNumber)
+	headerRepo := repositories.NewHeaderRepository(db)
+	headerID, headerErr := headerRepo.CreateOrUpdateHeader(fakeHeader)
+	Expect(headerErr).NotTo(HaveOccurred())
+
+	checkedHeaderRepo, repoErr := repositories.NewCheckedHeadersRepository(db, schema)
+	Expect(repoErr).NotTo(HaveOccurred())
+	uncheckedHeaders, uncheckedHeaderErr := checkedHeaderRepo.UncheckedHeaders(0, -1, 1)
```

Comment:
this is totally not your fault because it's how the function is implemented - but it might be cool to wrap `-1` (and potentially all of these numbers) in vars so that the intention is more expressive

Reply:
I might - although TBH I found some of these tests to be hard to follow because variables were in the setup and had to be understood at each level.

Maybe allowing a little WET in the tests would help with that.